graphql/util.go

```diff
@@ -367,7 +367,7 @@ func SchedulePatch(ctx context.Context, patchId string, version *model.Version,
 	// can't interrupt the db operations here
 	newCxt := context.Background()
 
-	projectRef, err := model.FindOneProjectRef(project.Identifier)
+	projectRef, err := model.FindMergedProjectRef(project.Identifier)
```

Comment:
Just for my clarity can you explain what scenarios call for returning a proj ref merged with repo settings vs. just getting the branch level settings?

Reply:
Thanks!